src/routes/api/comment.js

```diff
@@ -0,0 +1,30 @@
+import { Router } from 'express';
+import comments from '../../controllers/commentsController';
+import Validation from '../../middlewares/commentValidate';
+import getArticle from '../../middlewares/checkElements/checkArticle';
+import getComment from '../../middlewares/checkElements/checkComment';
+
+const app = Router();
+
+app.post(
+  '/:articleId/comments',
+  Validation,
+  getArticle,
+  comments.createComment
+);
+app.get('/:articleId/comments', getArticle, comments.getComments);
```

Comment:
use the approach we discussed.

Reply:
Do u mean to change the names? i.e. Instead of using getComment I give it the name of the file it is importing checkComment

Reply:
So I believe you don't need to get the article first then pass the results to `getCommets` you could getComments where Article_ID is `articleId `